readme.txt

```diff
@@ -66,19 +66,14 @@ To build extension for production run `./bin/build.sh`
 
 == Changelog ==
 
-= 4.0.2 =
```

Comment:
why Are you removing the `4.0.2` from the history ?

Reply:
No neither as the 4.0.0 and the 4.0.1